lib/rules/require-dollar-before-jquery-assignment.js

```diff
@@ -75,6 +75,41 @@ module.exports.prototype = {
             var varName;
             var right;
 
+            function checkIfVarNameShouldStartWithDollar(left, varName, right) {
+
+                if (varName.indexOf('$') === 0 || varName.indexOf('_$') === 0) {
+                    return;
+                }
+
+                if (!right || right.type !== 'CallExpression') {
```

Comment:
There is one test already that should cover this case:
```
it('should not report for loops', function() {
    expect(checker.checkString('for (var prop in rawVars) {}')).to.have.no.errors();
});
```
But I've added another that hopefully adds some clarity:
```
it('should not report for var declaration without assignment', function() {
    expect(checker.checkString('var x;')).to.have.no.errors();
});
```